src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -38,7 +38,8 @@ const OfflineKeyShareGeneration = () => {
     const notificationsStore: NotificationsStore = stores.Notifications;
     const { ownerNonce } = accountStore;
     const { accountAddress } = walletStore;
-    const { apiNetwork } = getCurrentNetwork();
+    const { apiNetwork, networkId } = getCurrentNetwork();
+    const isPrater = networkId !== NETWORKS.MAINNET;
```

Comment:
instead use reversed logic: `isNotMainnet`
Because we'll have prater and holesky together, and with new approach we won't need to change this place anymore

Reply:
done